lib/cli/ui/prompt.rb

```diff
@@ -91,8 +91,10 @@ def ask(question, options: nil, default: nil, is_file: nil, allow_empty: true, m
         # Confirmation question
         #   CLI::UI::Prompt.confirm('Is the sky blue?')
         #
-        def confirm(question)
-          ask_interactive(question, %w(yes no)) == 'yes'
+        #   CLI::UI::Prompt.confirm('Do a dangerous thing?', default_yes: false)
```

Comment:
I'd prefer a key of simply `default:`.  The `yes` in conjunction with the boolean, particularly when used as `default_yes: false` reads a bit weirdly, and since the return is boolean, this then becomes the default return value.

Reply:
Or a literal boolean like `default: true`? It feels counter intuitive to "leak" a `yes` string when the function doesn't take strings as arguments and returns a boolean.

Reply:
Yep, I want to keep the value a boolean, just change the name. 👍

Reply:
`default:` as a boolean sounds better to me too.  It keeps it consistent with ask as well.